searx/templates/simple/macros.html

```diff
@@ -12,6 +12,10 @@
     <img width="14" height="14" class="favicon" src="{{ url_for('static', filename='themes/simple/img/icons/' + favicon + '.png') }}" alt="{{ favicon }}" />
 {%- endmacro %}
 
+{% macro result_favicon_url(domain) -%}
+    https://www.google.com/s2/favicons?domain={{domain}}
```

Comment:
This request will lead in tracking the SearXNG users by google.

I think we need such a kind of service in SearXNG itself, a database that maps from domain to favicon.  To cite @dalf: 

[It is possible to make a side local server to host the favicons. 1 million websites x 1 Ko (avg size of a favicon) = 1Go. It is a lot of through the network, but locally it is not too much.](https://github.com/searxng/searxng/pull/441#issuecomment-953156937)